samples/04_gis_analysts_data_scientists/chennai_floods_analysis.ipynb

```diff
@@ -790,7 +790,7 @@
    "source": [
```

Comment:
<p>In the cell below, we will use a band combination [5,4,3] (a.k.a. mid-IR (Band 5), near-IR (Band 4) and red (Band 3)) of Landsat imagery to provide definition of land-water boundaries and highlight subtle details not readily apparent in the visible bands alone. We use more infrared bands to locate inland lakes and streams with greater precision. Generally, the wetter the soil, the darker it will appear, due to the infrared absorption capabilities of water.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>